src/mlpack/methods/ann/layer/add.hpp

```diff
@@ -97,6 +97,9 @@ class Add
   //! Modify the gradient.
   OutputDataType& Gradient() { return gradient; }
 
+  //! Get the output size.
+  size_t const& OutputSize() const { return outSize; }
```

Comment:
Can you use the same style as used here:

https://github.com/mlpack/mlpack/blob/master/src/mlpack/methods/ann/layer/linear.hpp#L119